### PR TITLE
always support fo4 dds archive, even on linux

### PR DESCRIPTION
**Pull request by Guekka**
With the recent progress on directxtex, it is now possible to use it on Linux

### CODE REVIEW

File: vcpkg.json [hunk 1]
```diff
@@ -16,12 +17,6 @@
         "zlib"
       ]
     },
-    "dds": {
-      "description": "Build support for dds files in ba2 archives",
-      "dependencies": [
-        "directxtex"
-      ]
-    },
```
Comment (Ryan-rsm-McKenzie) [inline vcpkg.json:24]: The presets file needs to reflect this feature being removed:
https://github.com/Ryan-rsm-McKenzie/bsa/blob/80f8e24b09ca35bac0b38b23ed8440d69b27ab84/CMakePresets.json#L51

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -94,27 +94,19 @@ find_package(binary_io REQUIRED CONFIG)
 find_package(LZ4 MODULE REQUIRED)
 find_package(mmio REQUIRED CONFIG)
 find_package(ZLIB MODULE REQUIRED)
+find_package(directxtex REQUIRED CONFIG)
```
Comment (Ryan-rsm-McKenzie) [inline src/CMakeLists.txt:97]: Can you order this alphabetically?

File: cmake/config.cmake.in [hunk 3]
```diff
@@ -3,10 +3,7 @@ include(CMakeFindDependencyMacro)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
 
-if(WIN32)
-	find_dependency(directxtex)
-endif()
-
+find_dependency(directxtex)
```
Comment (Ryan-rsm-McKenzie) [inline cmake/config.cmake.in:6]: Can you order this alphabetically?

**Review by Ryan-rsm-McKenzie — CHANGES_REQUESTED** [review]

**Comment by Guekka** [comment]
Thanks for your quick review, that's impressive

**Review by Ryan-rsm-McKenzie — APPROVED** [review]

**Pull request merged**